morse.rb

```diff
@@ -0,0 +1,28 @@
+def decode_char(mcode)
+  morsecode = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+                '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+                '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+                '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+                '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+                '--..' => 'Z', '-----' => '0', '.----' => '1', '..---' => '2', '...--' => '3',
+                '....-' => '4', '.....' => '5', '-....' => '6', '--...' => '7', '---..' => '8',
+                '----.' => '9', '.-.-.-' => '.', '--..--' => ',', '..--..' => '?' }
+  morsecode[mcode]
+end
+
+def decode_word(mword)
+  letters = mword.split
+  words = ''
+  letters.each do |letter|
+    words += decode_char(letter)
+  end
+  print "#{words} "
+end
```

Comment:
- [ ] Great work executing most of the commands. However, I noticed that you are printing the string representation of the _**Morse**_ word instead of returning it. Based on the requirements, you ought to:
- return the letter representation of the morse letter in the `decode_char` method which you did ✔️ 
- return the word representation of the morse word in the `decode_word` method which was not done ⛔ 
- return the sentence representation of the morse sentence in the `decode_message` method which was not done ⛔

Kindly return them and not print them. In the end, we would be able to do `print decode_char(...)` when testing the methods. 👍 

### Project's Requirement

![Screenshot_1](https://github.com/Luchy-B/morse-code/assets/77106962/7534cb0d-04ca-4786-b9d4-3d4e8755392d)